robot_tests/initrd.robot

```diff
@@ -3,7 +3,8 @@ Library  lib/Fakeroot.py
 Library  lib/Initrd.py
 Library  String
 Library  Collections
-Suite Setup  Setup
+Library  OperatingSystem
```

Comment:
Maybe to reduce the changes (i.e. adding `Initrd.`in front of a lot of keywords), using
```suggestion
Library  OperatingSystem  AS  OS
```
and then `OS.Set Environment Variable` would be better?